### DISCUSSION TITLE
V2024.02.1

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#60] [version]
●对话本在原有左右的基础上，新增一个居中（居中文字的背景色无法自定义）。
●优化部分页面细节，修复部分反馈过来的问题，提升性能和稳定性。
清单记本，长按发布按钮，可弹出批量发布对话框，方便单文本按行拆分后多条发布。
优化进展转移页面，单条进展显示行数增加到20行。

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/V2024.02.1'>V2024.02.1</a>.</em>

**Comment by undpunc** [comment]
可否考虑给清单计本添加一个到点提醒功能，比如在第二天发送通知和震动手机

然后希望可以隐藏打卡本里日历上的数字，和github的不一样有点奇怪

**Reply by WatsonYao** [reply]
提醒类的待办可能会出新的集子，清单只是简单的列表，没有提醒功能。
打卡本的日期显示，后续可能会添加一个配置项。

**Comment by buer333** [comment]
您好，非常喜欢nian这个app，在使用过程中发现了一个可以优化的地方，能否在文字类的记录中添加荧光笔背景，或者加粗斜体之类的，可以画一下重点，感谢。
还有一个问题就是，我在import的时候，总是显示未读取到数据，我的格式是这样的，也试过没有引号的格式，都是未读取到数据，请问是哪里出现问题了呢？手机是红米k40，安卓版本是11。
<img width="103" alt="屏幕截图 2024-03-02 212637" src="https://github.com/WatsonYao/nian_app/assets/82898320/6fe41250-356a-4d1e-92cc-3c2de6073a3e">

**Reply by buer333** [reply]
看了之前关宇import的帖子，也都按照那个帖子的方法进行的，用电脑复制进去的。

**Reply by WatsonYao** [reply]
用系统文件管理器查看属性，是否是全局可读写的。
我自己也用的K40，测试OK的。不行用系统文件管理器复制再粘贴文件试试。
进展采用纯文本，不支持背景粗细等样式。

**Reply by buer333** [reply]
是可读写的，也复制粘贴了，依旧无法解决导入问题。

**Reply by buer333** [reply]
您好，找到问题了，日期与日期之间需要有个回车，感谢。
